apiary.apib

```diff
@@ -699,7 +699,7 @@ Get details for a single contact.
                         + id: `40157f9d-1322-4f2e-b245-188efdc52cc2` (string)
             + language: `en` (string)
             + payment_term (PaymentTerm, nullable)
-            + remarks: `First contact at expo` (string)
+            + remarks: `First contact at expo` (string) - Formatted in markdown
```

Comment:
I've been using ` - Uses markdown formatting` in my other PR. Maybe we should make it consistent?